src/txn/txn_rollback_to_stable.c

```diff
@@ -271,7 +271,7 @@ __rollback_row_ondisk_fixup_key(WT_SESSION_IMPL *session, WT_PAGE *page, WT_ROW
           __wt_timestamp_to_string(hs_stop_ts, ts_string[2]),
           __wt_timestamp_to_string(rollback_timestamp, ts_string[3]));
 
-        newer_hs_ts = hs_start_ts;
+        newer_hs_ts = durable_ts;
```

Comment:
This change/fix looks like it not related to prepare...?  If that's true, it's fine, let's just make sure it's listed in the squashed commit comment.

Reply:
Added comment and will make sure that it is mentioned in the commit.